spec/features/users/new_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe 'As a visitor' do
+  describe 'When I click the Register link on the Welcome Page' do
+    before :each do
+      User.create!(name: "Tim", email: "tim@gmail.com", password: "test")
+
+      visit root_path
+
+      click_link "New to Viewing Party? Register Here"
+    end
+
+    it "I am taken to the registration page where there is a form to register" do
+      expect(current_path).to eq('/register')
+      expect(page).to have_content("Email")
+      expect(page).to have_content("Password")
+      expect(page).to have_content("Confirm Password")
+      expect(page).to have_button("Register")
+      expect(page).to have_link("Already Registered? Log in Here")
+    end
+
+    it "creates a new user who should be then logged in and redirected to dashboard page" do
+
+      fill_in :name, with: "Brian"
+      fill_in :email, with: "brian@gmail.com"
+      fill_in :password, with: "password"
+      fill_in :password_confirmation, with: "password"
+
+      click_button "Register"
+
+      user = User.last
+
+      expect(user.name).to eq("Brian")
```

Comment:
I think we decided that line 31 and 33 were not needed. however it also isn't an issue.